templates/web/map/map.html

```diff
@@ -0,0 +1,139 @@
+<div id="map"></div>
+<script>
+mapboxgl.accessToken = 'pk.eyJ1Ijoicm9hbGRzdG9ybSIsImEiOiJjbHdkaDhlN2QxNGFvMnFwcDB6YmFjd3RsIn0.kZAP04DN43268ORUH5bJsw';
```

Comment:
This needs to be parameterized, and maybe just disable the map functionality all together if the token is not supplied.